TestBots/TestSpadesBot.cs

```diff
@@ -25,6 +25,55 @@ public void CountSureTricks()
             Assert.AreEqual("QS", suggestion.ToString(), $"Suggested {suggestion.StdNotation}; expected QS");
         }
 
+        [TestMethod]
+        [DataRow("2H3H", "JH5H", "JH")]
+        [DataRow("3H2H", "JH5H", "JH")]
+        [DataRow("9HTH", "QHJH", "JH")]
+        [DataRow("9HTH", "KHJH", "KH")]
+        [DataRow("TH9H", "QH7H", "QH")]
+        [DataRow("TH9H", "JH7H", "7H")]
```

Comment:
Add a version of this test where the opponent is taking the trick with `TH` (because then we should play `JH`).